src/components/FloatingActionButton.tsx

```diff
@@ -95,38 +96,46 @@ function FloatingActionButton({onPress, isActive, accessibilityLabel, role}: Flo
         Platform.OS === 'web' ? undefined : adapter,
     );
 
+    const toggleFabAction = (event: GestureResponderEvent | KeyboardEvent | undefined) => {
+        // Drop focus to avoid blue focus ring.
+        fabPressable.current?.blur();
+        onPress(event);
+    };
+
     return (
         <Tooltip text={translate('common.new')}>
-            <View style={styles.bottomTabBarItem}>
-                <AnimatedPressable
-                    ref={(el) => {
-                        fabPressable.current = el;
-                        if (buttonRef && 'current' in buttonRef) {
-                            buttonRef.current = el;
-                        }
-                    }}
-                    accessibilityLabel={accessibilityLabel}
-                    role={role}
-                    pressDimmingValue={1}
-                    onPress={(e) => {
-                        // Drop focus to avoid blue focus ring.
-                        fabPressable.current?.blur();
-                        onPress(e);
-                    }}
-                    onLongPress={() => {}}
-                    style={[styles.floatingActionButton, animatedStyle]}
-                >
-                    <Svg
-                        width={variables.iconSizeNormal}
-                        height={variables.iconSizeNormal}
+            <PressableWithoutFeedback
+                style={styles.h100}
+                accessibilityLabel={accessibilityLabel}
+                onPress={toggleFabAction}
+            >
```

Comment:
Why we are adding another Pressable?

Reply:
I think it is because we want to increase the "clickable" area of the button, to improve UX, especially on the mobile.
cc @MaciejSWM

Reply:
I also have the same question. why can't we use padding or some other styling to increase the pressable area?

I don't think it's a good idea to have two Pressables with the same callback.